reana_workflow_engine_yadage/externalbackend.py

```diff
@@ -106,7 +106,7 @@ def submit(self, spec, parameters, state, metadata):
         log.info('submitting!')
 
         job_id = submit.submit(
-            os.getenv('REANA_WORKFLOW_ENGINE_YADAGE_EXPERIMENT', 'default'),
+            os.getenv('REANA_WORKFLOW_ENGINE_EXPERIMENT', 'default'),
```

Comment:
This component is called `reana-workflow-engine-yadage`, so the environment variable `REANA_WORKFLOW_ENGINE_YADAGE_...` seems appropriately named?

(Otherwise we could cross this with the concept of "organisations" defined elsewhere...)

Reply:
as far as I remember, it was changed to make the `workers-template` more generic: https://github.com/reanahub/reana-resources-k8s/pull/28#discussion_r142996332